test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -86,7 +87,8 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 		}
 
 		// we expect having the same policy across all NRTs
-		tmPolicy = nrts[0].TopologyPolicies[0]
+		tps := noderesourcetopology.GetTopologyPolicies(nrts[0])
+		tmPolicy = tps[0]
```

Comment:
again just fetch the first attribute, possibly erroring out if there are duplicate attrs (which is again legal but uncommon/unexpected). The duplicated attributes check can probably generalized/made smarter